ingestion/src/metadata/ingestion/source/database/sql_column_handler.py

```diff
@@ -0,0 +1,286 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Generic call to handle table columns for sql connectors.
+"""
+import re
+import traceback
+from typing import List, Optional, Tuple
+
+from sqlalchemy.engine.reflection import Inspector
+
+from metadata.generated.schema.api.tags.createTag import CreateTagRequest
+from metadata.generated.schema.entity.data.table import (
+    Column,
+    Constraint,
+    ConstraintType,
+    DataType,
+    TableConstraint,
+)
+from metadata.generated.schema.entity.tags.tagCategory import Tag
+from metadata.generated.schema.type.tagLabel import TagLabel
+from metadata.ingestion.ometa.client import APIError
+from metadata.ingestion.ometa.utils import ometa_logger
+from metadata.utils.column_type_parser import ColumnTypeParser
+from metadata.utils.fqdn_generator import get_fqdn
+
+logger = ometa_logger()
+
+
+class SqlColumnHandler:
+    def fetch_tags(self, schema: str, table_name: str, column_name: str = "") -> List:
+        return []
```

Comment:
Otherwise, we miss data without noticing. As `fetch_tags` is not marked as abstract, nothing happens here and there is no help from the interpreter